src/main/java/com/interfaceict/ptrakcer/controllers/ProjectEntityController.java

```diff
@@ -0,0 +1,32 @@
+//
+// Created by htrap19 on 1/8/22
+//
+
+package com.interfaceict.ptrakcer.controllers;
+
+import com.interfaceict.ptrakcer.models.ProjectEntity;
+import com.interfaceict.ptrakcer.servicesimpl.ProjectEntityServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+public class ProjectEntityController
+{
+    @Autowired
+    private ProjectEntityServiceImpl m_Service;
+
+    @GetMapping("/projects")
+    public List<ProjectEntity> getAllProjects() { return m_Service.getAll(); }
+
+    @PostMapping("/projects")
+    public void newProject(@Valid @RequestBody ProjectEntity newProject) { m_Service.save(newProject); }
```

Comment:
You could use a simple **POJO** instead of the real entity **POJO**

Reply:
Also can we use DTO ?